packages/cli/src/index.ts

```diff
@@ -132,6 +133,15 @@ cli.command('create [appName]', 'create a new marketplace app').action(async (ap
         },
         {
             type: 'select',
+            name: 'appType',
+            message: 'Select the type of your app',
+            choices: [
+                { title: 'App (alpha)', value: 'platformApp' },
```

Comment:
Can you put it behind a env flag (like `EXPERIMENTAL_PLATFORM_APP`)?

Reply:
Ah perfect, do you have an example where it is already used?

Reply:
How about adding a -e --experimental flag to the command?